src/settings.ts

```diff
@@ -74,57 +74,55 @@ export class Settings {
    * @return {string} The bearer token.
    */
   public static getBearerToken(): string {
-    return Settings.readFileContent("~/.cloco/config/cloco_token");
+    return Settings.readFileContent(`${process.env.HOME}/.cloco/config/cloco_token`);
```

Comment:
I thought MS joined proper IT and finally got themselves some bash? Presumably that means cloco-bash will work in that context?